Samples/CSharp/CSharpSamples.ruleset

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this file? `.editorconfig` can now set the action info/warning/error in VS.

It can't do it on build from the CLI though, although there is a new analyzer in the works that makes use of the `.editorconfig` and presumably provide fixes too.

Reply:
More info here:

https://www.strathweb.com/2020/01/enforcing-c-editorconfig-formatting-conventions-at-build-time/

Perhaps we should add a new Ci build that makes use of this analyser to make sure we haven't broken anything?

Reply:
That's would be awesome!